compiler/assemble.js

```diff
@@ -174,6 +174,8 @@ export default (funcs, globals, tags, pages, data, flags, noTreeshake = false) =
       if (func.constr) argc -= 4;
       if (!func.internal || func.typedParams) argc = Math.floor(argc / 2);
 
+      if (name.startsWith('#porf')) name = '';
```

Comment:
Could we just use a `#` prefix instead?

Reply:
Won't that break private methods whenever we implement them?

Reply:
The name wouldn't begin with `#` internally and wouldn't be directly in the function name anyway? `#` is just used for parsing, there is a separate private identifier AST type/flag

Reply:
Well yes, but at least the .name attribute does include the hash. Regardless, I can change it if you'd like.

Reply:
The function name internally could not just be that since there could be multiple classes with the same private method name, so there would have to be a prefix of the class name or something before regardless.

Reply:
True, true. I'll change it, o7.